src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDeployment.java

```diff
@@ -139,6 +139,7 @@ void deployConfig(String filename, DeployProjectConfigurationConfiguration confi
     List<String> arguments = new ArrayList<>();
     arguments.add("deploy");
     arguments.add(deployable.toAbsolutePath().toString());
+    arguments.addAll(GcloudArgs.get("server", configuration.getServer()));
```

Comment:
what if this is not set, i.e. null?

Reply:
GcloudArgs#get [calls](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/a028c181b7296148fb5d213c171f6ad475e31b55/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/GcloudArgs.java#L31) [Args#string](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/a028c181b7296148fb5d213c171f6ad475e31b55/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java#L37), which only adds the parameter if not null.